io.sloeber.core/src/io/sloeber/core/tools/uploaders/arduinoUploader.java

```diff
@@ -111,20 +104,16 @@ public boolean uploadUsingPreferences(IFile hexFile, boolean usingProgrammer, IP
 	 */
 	@SuppressWarnings("nls")
 	private String getPasswordFromCode() {
-		return IndexHelper.findParameterInFunction(this.myProject, "setup", "ArduinoOTA.setPassword",
+		String parameter = IndexHelper.findParameterInFunction(this.myProject, "setup", "ArduinoOTA.setPassword",
```

Comment:
But why?! I just spent one hour trying to figure out why it was not using the password I entered in the port settings. In my code I had a variable which contains the password - it used the variable name as password which did not work.

This is not very user friendly. Please allow us to just specify the password when uploading. Also why forbid changing the password via OTA?